de.po

```diff
@@ -2315,19 +2315,19 @@ msgstr "Benutzt von"
 #: src/components/create-vm-dialog/createVmDialog.jsx:539
 #, fuzzy
 msgid "User login"
-msgstr "Letzte Anmeldung"
+msgstr "Benutzer Anmeldung"
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:187
 msgid "User login must not be empty when user password is set"
-msgstr ""
+msgstr "Benutzer Login darf nicht leer sein wenn Passwort gesetzt ist"
```

Comment:
Please use a consistent wording here, same as "User login" above. "Benutzername" is our standard term in cockpit.